### ISSUE TITLE
format, cleanup, sort

### ISSUE THREAD

**Issue by tehn** [monome/norns-community#245]
a few of us are working towards migrating the `norns.community` wiki to a much lighter, automatically generated static site. the primary goal is to lower the workload for contributors and avoid some of the maintenance issues we've had with the current overspec'd wikijs system.

the current progress on the static site generator is here: https://github.com/monome-community/norns-community

TODO to facilitate the above

1. we need to standardize the "author" field. currently it's (mostly) of the form `"Brian Crabtree (tehn)"`. we need this to be just `"tehn"`.
2. some author tags are collaborations. these need to be rewritten as a JSON list ie `["tehn", "zebra"]`
3. the full catalog needs to be run through a sort by `project_name`

(the reason we're doing this is to make url-friendly author names, to facilitate seeing a list of scripts written by an author)

**Comment by p3r7** [comment]
done i guess https://gist.github.com/p3r7/881def574c80d9063f983b7ebc823aca

**Comment by ngwese** [comment]
one recommendation i would also make with respect to site generation is that the sorting of entries be performed by the site generation process as opposed to requiring the catalog to be sorted. since the catalog is hand edited maintaining sort order is not enforced. the maiden code sorts catalog entries (by name) before presenting.

**Comment by tehn** [comment]
agreed, i don't think we can expect the entries to be sorted and it's easy to do after importing.
